CONTRIBUTING.md

```diff
@@ -73,3 +74,5 @@ If your contribution lacks any of these things, they will have to be added
 by a core contributor before being merged into django-import-export proper, which may take
 substantial time for the all-volunteer team to get to.
 
+
+[coc]: https://github.com/bmihelac/django-import-export/blob/master/CODE_OF_CONDUCT.md
```

Comment:
Shouldn't this be `https://github.com/django-import-export/django-import-export/blob/master/CODE_OF_CONDUCT.md` instead?

Reply:
Thanks, fixed